lib/bugsnag/logger.ex

```diff
@@ -2,39 +2,41 @@ defmodule Bugsnag.Logger do
   require Bugsnag
   require Logger
 
-  use GenEvent
+  @behaviour :gen_event
 
-  def init(_mod, []), do: {:ok, []}
+  def init(_init_args), do: {:ok, []}
 
   def handle_call({:configure, new_keys}, _state) do
     {:ok, :ok, new_keys}
   end
 
   def handle_event({_level, gl, _event}, state)
-  when node(gl) != node() do
+      when node(gl) != node() do
```

Comment:
For now, could we avoid reformatting existing code in the context of other changes? I'd like to do a `mix format` pass, but I think it should be a separate task/commit that formats the whole codebase and integrates a CI check so formatting can be automatically enforced from that point on.

Reply:
Hmm, i've auto-format on save.. plus since its a standard formatter - what can be the damage?

Reply:
BTW, i dont mind running mix format as part of this PR.
Actually, let me do just that and we can move behind this

Reply:
Please remove the formatting changes from this PR — I don't mind if you'd like to contribute this, but I'd prefer it to be separate so it can be reviewed and discussed without holding up unrelated changes.